clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1496,4 +1496,32 @@ public static String replaceSuffix(String str, String oldSuffix, String newSuffi
             throw new IllegalArgumentException("Expected string to end with " + oldSuffix + " but string is " + str);
         return str.substring(0, str.length() - oldSuffix.length()) + newSuffix;
     }
+
+    public static long zeroIfNegative(long value) {
+        return Math.max(0L, value);
+    }
+
+    // returns the sum of a and b unless it would overflow, which will return Long.MAX_VALUE
```

Comment:
Any reason for this not to be a Javadoc?

Reply:
Updated with javadoc. Thanks.